crates/swc_ecma_minifier/src/compress/optimize/iife.rs

```diff
@@ -322,6 +296,96 @@ where
         clean_params(callee);
     }
 
+    /// If a parameter is not used, we can ignore return value of the
+    /// corresponding argument.
+    pub(super) fn ignore_unused_args_of_iife(&mut self, e: &mut CallExpr) {
+        let has_spread_arg = e.args.iter().any(|v| v.spread.is_some());
+        if has_spread_arg {
+            return;
+        }
```

Comment:
Is this necessary? I'm assuming this is because we can't appropriately track the index position of the call arg and the function param after a spread, but we could eliminate args before the spread.

Reply:
I think it would be better, will do it tomorrow.